### PR TITLE
Remove politics 

### PR DESCRIPTION
**Pull request by nudelx**
Make Code Not War !!!

### CODE REVIEW

**Review by marat-asadurian — APPROVED** [review]

**Review by orenk86 — APPROVED** [review]

**Review by olegsu — APPROVED** [review]

**Review by elie222 — APPROVED** [review]

**Review by moriatuch — APPROVED** [review]

**Review by liorgreenb — APPROVED** [review]

**Review by RomH1 — APPROVED** [review]

**Review by david-a — APPROVED** [review]
Long live Devland!

**Review by nitzanmo — APPROVED** [review]

**Review by RLArt7 — APPROVED** [review]
We Already Free!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!-- @format -->
-> # Black Lives Matter 🖤 Free Palestine 🇵🇸
+> # Black Lives Matter 🖤 Make Code Not War ♥️ 🌿 🌏
```
Comment (Attrash-Islam) [inline README.md:2]: Why Palestine is a war while Israel is the heaven? 😆  I don't think you're living in the world

Comment (elie222) [reply README.md:2]: The change doesn't mention Israel or Palestine 🤨

Comment (Attrash-Islam) [reply README.md:2]: When you change something to another thing you can't detach the context. Why he thought about War then? If it was a clean change I'd definitely agree with you. Try to think about it and simulate the change, anyone can infer what I'm referring to.

Comment (elie222) [reply README.md:2]: He didn't say anything about Israel heaven or Palestine war that you mentioned in your original comment.

As hundreds of other people have already mentioned it should never have entered the repo to begin with.

**Comment by leebyron** [comment]
The GraphQL technical steering committee is working on a policy to guide the use of political speech from the voice of a GraphQL project. The original author chose to revert the original change via #1879 until we complete that policy. This should resolve this issue, so I am closing it.

As a reminder, all discussion within GraphQL projects must adhere to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md).

**Pull request closed without merging**